crates/nu-command/src/bytes/starts_with.rs

```diff
@@ -60,13 +61,73 @@ impl Command for BytesStartsWith {
             pattern,
             cell_paths,
         };
-        operate(
-            starts_with,
-            arg,
-            input,
-            call.head,
-            engine_state.ctrlc.clone(),
-        )
+
+        match input {
+            PipelineData::ExternalStream { stdout, span, .. } if stdout.is_some() => {
+                let stream = stdout.expect("stdout is some");
```

Comment:
```suggestion
        match input {
            PipelineData::ExternalStream { stdout: Some(stream), span, .. } => {
```

Reply:
Excellent pattern matching, this makes me feel much better about things. Thanks.